app/components/maintenance-banner/component.js

```diff
@@ -15,6 +21,11 @@ export default Ember.Component.extend({
 
     hidden: 'true',
 
+    alert: true,
+    _info: true,
```

Comment:
Why not use a single bound property that's a string? The you can just do something like:
```
classNameBindings: ['alertClass'],

alertClass: 'alert-info',
.
.
.
this.set('alertClass', `alert-${levelMap[maintenance.level]}`);
```

Reply:
Also, you can use classNames and classNameBindings simultaneously so you can put `alert` in classNames without needing a bound property.